src/middleware/with-jwt-verified.ts

```diff
@@ -5,71 +5,105 @@ import { Response } from "retes/response";
 import { SALEOR_AUTHORIZATION_BEARER_HEADER } from "../const";
 import { getSaleorHeaders } from "../headers";
 import { getJwksUrl } from "../urls";
+import { createMiddlewareDebug } from "./middleware-debug";
+
+const debug = createMiddlewareDebug("withJWTVerified");
 
 export interface DashboardTokenPayload extends jose.JWTPayload {
   app: string;
 }
 
+const ERROR_MESSAGE = "JWT verification failed:";
+
 export const withJWTVerified =
   (getAppId: (request: Request) => Promise<string | undefined>): Middleware =>
   (handler) =>
   async (request) => {
     const { domain, authorizationBearer: token } = getSaleorHeaders(request.headers);
-    const ERROR_MESSAGE = "JWT verification failed:";
 
-    if (token === undefined) {
+    debug("Middleware called with domain: \"%s\"", domain);
+
+    if (typeof token !== "string") {
+      debug("Middleware with empty token, will response with Bad Request", token);
+
       return Response.BadRequest({
         success: false,
         message: `${ERROR_MESSAGE} Missing ${SALEOR_AUTHORIZATION_BEARER_HEADER} header.`,
       });
     }
 
+    debug("Middleware called with token starting with: \"%s\"", token.substring(0, 4));
+
     let tokenClaims: DashboardTokenPayload;
+
     try {
       tokenClaims = jose.decodeJwt(token as string) as DashboardTokenPayload;
+      debug("tokenClaims decoded from jwt");
```

Comment:
```suggestion
      debug("TokenClaims decoded from JWT");
```

Reply:
Hmm, `tokenClaims` mimics variable name, `TokenClaims` is something that doesn't "exist". So maybe `Token claims` and detach from variable here?

Reply:
Sounds good!